exonum/src/runtime/rust/mod.rs

```diff
@@ -61,10 +61,10 @@
 //!     WalletAlreadyExists = 1,
 //! }
 //!
-//! // Define the transaction interface for your service by creating a trait with
+//! // Define a transaction interface for your service by creating a `Transactions` trait with
 //! // the following attribute and method signatures.
-//! // This attribute implements `Interface` trait for this trait and `Transaction`
-//! // trait for each argument.
+//! // This attribute implements the `Interface` trait for the `Transactions` trait and a `Transaction`
```

Comment:
Nit: This was already in the text, but "implementing a trait for trait" looks super-weird from the Rust perspective. (In reality, `Interface` is implemented for a trait *object* based on the `Transactions` trait - `impl Interface for dyn Transactions`.)